packages/app/src/cli/services/app-logs/poll-app-logs.ts

```diff
@@ -100,17 +100,30 @@ export const pollAppLogs = async ({
 
             const logs = payload.logs
             if (logs.length > 0) {
-              stdout.write(logs)
+              stdout.write(
+                logs
+                  .split('\n')
+                  .filter(Boolean)
+                  .map((line: string) => outputContent`${outputToken.gray('│ ')}${line}`.value)
+                  .join('\n'),
+              )
             }
           } else {
             stdout.write(JSON.stringify(payload))
           }
 
-          await writeAppLogsToFile({
+          const logPath = await writeAppLogsToFile({
             appLog: log,
             apiKey,
             stdout,
           })
+          stdout.write(
+            outputContent`${outputToken.gray('└ ')}${outputToken.link(
+              'Open log file',
+              `file://${logPath}`,
```

Comment:
re: Windows -- might need to check that these file paths are valid there. e.g. if `logPath` is like `C:\something`.

Reply:
I discovered that the forward slashes are actually due to our use of [`pathe`](https://www.npmjs.com/package/pathe), which presupposes that forward slashes are OK in Windows these days. Which I think for our use case, that would be true. So even w/ hyperlinks disabled, we are outputting forward slashes at the moment.